cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java

```diff
@@ -218,9 +218,10 @@ public void execute() {
             targetPath = sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
             CommandUtil.printError(this.errStream,
-                    "invalid Ballerina source path, it should either be a module name in a Ballerina project or a " +
-                    "file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension.",
-                    "ballerina run {<bal-file> | <module-name>}",
+                    "invalid Ballerina source path. It should either be a name of a module in a Ballerina project, " +
+                    "a file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension, or an executable .jar " +
```

Comment:
.bal and .jar are inconsistent here right?

```suggestion
                    "a file with a \'" + BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension, or an executable '.jar' " +
```

Reply:
Fixed